transports/http/endpoints/api/merkleroots/model.go

```diff
@@ -0,0 +1,42 @@
+package merkleroots
+
+import (
+	"github.com/libsv/bitcoin-hc/domains"
+)
+
+// merkleRootConfirmationRespose is an API response for
+// confirmation of merkle roots inclusion in the longest chain.
+type merkleRootConfirmationRespose struct {
```

Comment:
or maybe it could be:
```json
{
  "confirmed": true, // true if all are confirmed otherwise false
  "headers": [ {
        "merkleroot":"",
        "hash": "",
        "height": "",
        "version": "",
        "nonce": "",
        "bits": "",
        "chainwork": "",
        "previousblock": "",
        "timestamp": "",
        "cumulatedWork": "",
        "header_state": "",
        "confirmed": true
  } ]
}
```

and then even change the path to something to something like `/chain/merkleroot/search`

Reply:
I agree that the second idea was bad.
And about `allIncluded` I added another comment that maybe `allConfirmed` is a better (more consistent) name.